pages/integrations/github.md.erb

```diff
@@ -1,81 +1,82 @@
 # GitHub
 
-Buildkite can connect to your GitHub repository and use the Commit Status API to update the status of commits in Pull Requests.
+Buildkite can connect to your GitHub repository and use the [Commit Status API](https://docs.github.com/en/rest/reference/repos#statuses) to update the status of commits in pull requests.
 
-To complete this integration, you will need admin privileges for your GitHub repository.
+To complete this integration, you need admin privileges for your GitHub repository.
 
 {:toc}
 
 ## Connecting Buildkite and GitHub
```

Comment:
Some of these probably need `Step 1` adding I think?

Reply:
Which ones? We have:

Connecting Buildkite and GitHub - introductory
Connect your Buildkite account to GitHub using the GitHub App - method 1
Connect your Buildkite account to GitHub using OAuth - method 2
Set up a new pipeline for a GitHub repository - this is step 2 of either of the methods above. Which one do I position it under? Alternatively, we could create a new heading ("pipeline setup tutorial"?) and make this step 1. But to have step 1/2/3 simply appear, at the same level of a bunch of other sections, in the middle of the article, really doesn't make much sense? Even if flagged as a tutorial with another heading. (the toc restrictions are really painful when you get into longer documents like this)
Branch configuration and settings - 3, see 2
Running builds on pull requests  - 4, see 2
Noreply email handling - standalone
Using one repository in multiple pipelines and organizations - standalone

I certainly don't like the flow of this document - it would be better if we could create clearer sections. But I think just turning some of them into steps in a largely invisible "tutorial", with a bunch of reference material surrounding it, adds confusion.

Reply:
Yeah, makes sense, we can leave it for now I think.